python/pyspark/ml/tests.py

```diff
@@ -844,6 +844,28 @@ def test_string_indexer_from_labels(self):
             .select(model_default.getOrDefault(model_default.outputCol)).collect()
         self.assertEqual(len(transformed_list), 5)
 
+    def test_vector_size_hint(self):
```

Comment:
This test doesn't test the 2 functions which were buggy.  Could you please test those (and simplify the test if possible)?  Thanks!

Reply:
@jkbradley Sorry, my bad. I added set/getSize and removed VectorAssembler from the test to simply.

Reply:
Thanks!  FYI this still isn't really testing the return value of setSize, but I think it's OK since we don't really do that anywhere else : P and I'm confident in the above change.